main.rb

```diff
@@ -0,0 +1,141 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './classroom'
+require './student'
```

Comment:
- [ ] In `student.rb` the `attr_reader` is created outside the class this will raise an error, in order to fix it the `attr_accessor` methods should be created inside the `class`